examples/Spirit/spirit_utils.h

```diff
@@ -226,11 +226,23 @@ double calcWork(
 ///     @param x_trajs a vector of the state trajectory for each mode
 ///     @param u_traj the control trajectory
 template <typename T>
-double calcWork(
+double calcMechanicalWork(
     drake::multibody::MultibodyPlant<T> & plant,
     std::vector<drake::trajectories::PiecewisePolynomial<double>>& x_trajs,
     drake::trajectories::PiecewisePolynomial<double>& u_traj);
 
+/// Calculates the electrical work done during trajectory, handles discontinuities
+///     @param plont, a pointer to the robot's model
+///     @param x_trajs a vector of the state trajectory for each mode
+///     @param u_traj the control trajectory
+///     @param efficiency, gain on what percent of negative power is useable by the battery
+template <typename T>
+double calcElectricalWork(
+    drake::multibody::MultibodyPlant<T> & plant,
+    std::vector<drake::trajectories::PiecewisePolynomial<double>>& x_trajs,
+    drake::trajectories::PiecewisePolynomial<double>& u_traj,
+    double efficiency = 0);
```

Comment:
the name efficiency confused me til I read the comment don't have a better name though. 
backEfficiency?
regenEfficiency?